stdlib/public/core/Unicode.swift

```diff
@@ -276,6 +262,9 @@ extension Unicode.UTF8: UnicodeCodec {
     if _fastPath(s == 0) { return }
     processCodeUnit(UInt8(truncatingIfNeeded: s) &- 0x01)
   }
+}
+
+extension Unicode.UTF8 {
```

Comment:
The non-deprecated APIs in this extension could be moved into UTF8.swift:

* `Unicode.UTF8.isContinuation(_:)`
* `Unicode.UTF8._nullCodeUnitOffset(in:)`